docs/06-concepts/01-working-with-endpoints.md

```diff
@@ -75,11 +75,10 @@ The return type must be a typed Future. Supported return types are the same as f
 
 ## Ignore endpoint definition
 
-If you want the code generator to ignore an endpoint definition, you can annotate the class with `@ignoreEndpoint`, imported from `serverpod_shared/annotations.dart`.  This can be useful if you want to keep the definition in your codebase without generating server or client bindings for it.
```

Comment:
Removed the extra import here and in the examples for now, as my current feeling is that we'll re-export the annotations (due to the alternatives not being clear wins for now, maybe something to re-think in v3 if package dependencies should change).